docs/local-procedures/Class C/brisbane.md

```diff
@@ -49,8 +49,7 @@ When the ATIS includes `ALL DEPARTURES MUST REQUEST PUSH BACK ON 118.85`, all pi
 
 !!! phraseology
     **QFA291:** "Brisbane Delivery, QFA291, bay 32, request pushback"  
-    **BN ACD:** "QFA291, estimated delay 10 minutes, remain this frequency."  
-    **QFA291**: "Remain this frequency, QFA291"  
+    **BN ACD:** "QFA291, standby, estimated delay 10 minutes"  
```

Comment:
```suggestion
    **BN ACD:** "QFA291, estimated delay 10 minutes, remain this frequency."  
    **QFA291**: "Remain this frequency, QFA291"
```

Reply:
The 'remain this frequency' instruction is used in this scenario to ensure pilots don't switch to SMC until directly advised